examples/tutorial/02-playlists/03-advanced-examples/01-duplicate-a-complete-playlist.js

```diff
@@ -0,0 +1,88 @@
+/*
+
+    Run with:
+
+        export SPOTIFY_ACCESS_TOKEN="<Token content here>"
+        node example/folder/file.js <Playlist Name>
+
+    Playlist Name defaults to 'Test' if not provided.
+
+*/
+const https = require('https');
+const SpotifyWebApi = require('../../../../');
+const SpotifyWebApiTools = require('../../spotify-web-api-tools.js');
+
+const spotifyApi = new SpotifyWebApi();
+const swat = new SpotifyWebApiTools(spotifyApi);
+
+spotifyApi.setAccessToken(process.env.SPOTIFY_ACCESS_TOKEN);
+
+const playlistName = process.argv.slice(2)[0] || 'Test';
+
+(async () => {
+  // Find Playlist by Name
+  const playlist = await swat.findUserPlaylistByName(playlistName);
+  console.log(`Using Playlist '${playlist.name}' with id '${playlist.id}'`);
+
+  // Get Tracks from old Playlist and filter local tracks
+  const tracks = (await swat.getAllPlaylistTracks(playlist.id)).filter(t => !t.is_local);
+  console.log(`Found ${tracks.length} tracks in Playlist`);
+
+  // Get Cover Image
+  const coverImageUrl = playlist.images[0].url;
+  const coverImageBase64 = await getImageFromUrlAsBase64(coverImageUrl);
+
+  // Create new Playlist
+  const newPlaylist = await spotifyApi.createPlaylist(
+    playlistName + ' (duplicate)',
+    {
+      public: false,
+      description: playlist.description
+    }
+  );
+  const newPlaylistId = newPlaylist.body.id;
+  console.log(
+    `Created Playlist '${newPlaylist.body.name}' with id '${newPlaylistId}'`
+  );
+
+  // Add Tracks to new Playlist (add max. 100 Tracks per Request according do API Spec)
```

Comment:
Hey, I was just randomly looking through the PR, since I've been meaning to contribute to the wrapper as well. Nothing important but just wanted to say "according do" should be "according to" I guess? :)

Reply:
Yeah sure, I didn't run the comments through deepl.com or something similar to double check😅